lightning/src/util/events.rs

```diff
@@ -126,7 +126,8 @@ pub enum Event {
 		/// now + 5*time_forwardable).
 		time_forwardable: Duration,
 	},
-	/// Used to indicate that an output was generated on-chain which you should know how to spend.
+	/// Used to indicate that an output which you should know how to spend which was confirmed
```

Comment:
Grammar-wise, consider rephrasing as:

Used to indicate that an output which you should know how to spend was confirmed on-chain and is now spendable.